test/test_pages/post_request_ajax.html

```diff
@@ -8,14 +8,18 @@
     <script type="text/javascript" src="shared/utils.js"></script>
     <script type="application/javascript">
 
+      function do_xhr(data) {
+        var xhr = new XMLHttpRequest;
```

Comment:
This should be `new XMLHttpRequest()`

Reply:
That sounds much better. Can you update your branch so that the Travis-CI tests will run again?

Reply:
Fixed this and triggered a re-run of the tests.